tests/system/action/mediafile/test_create_directory.py

```diff
@@ -0,0 +1,303 @@
+from tests.system.action.base import BaseActionTestCase
+
+
+class MediafileCreateDirectoryActionTest(BaseActionTestCase):
+    def test_create_directory_correct(self) -> None:
+        self.create_model(
+            "group/7", {"name": "group_LxAHErRs", "user_ids": [], "meeting_id": 110}
+        )
+        self.create_model("meeting/110", {"name": "meeting110"})
+        response = self.client.post(
+            "/",
+            json=[
+                {
+                    "action": "mediafile.create_directory",
+                    "data": [
+                        {
+                            "meeting_id": 110,
+                            "title": "title_Xcdfgee",
+                            "access_group_ids": [7],
+                        }
+                    ],
+                }
+            ],
+        )
+        self.assert_status_code(response, 200)
+        model = self.get_model("mediafile/1")
+        assert model.get("title") == "title_Xcdfgee"
+        assert model.get("access_group_ids") == [7]
+        assert model.get("is_directory") is True
+
+    def test_create_directory_parent(self) -> None:
+        self.create_model(
+            "group/7", {"name": "group_LxAHErRs", "user_ids": [], "meeting_id": 110}
+        )
+        self.create_model("meeting/110", {"name": "meeting110"})
+        self.create_model(
+            "mediafile/110",
+            {"title": "title_srtgb199", "child_ids": [111], "meeting_id": 110},
```

Comment:
Why do you set `child_ids` here? This should be set automatically by the relation handling

Reply:
I think this is correct here. The creation in favor of a test is not run through any relation handling code.

Reply:
Yes, I know, but if this is needed here, there is a bug in the action since this situation can never happen. If this is not needed, it can be removed.